backend/app/ownership_request/routes.py

```diff
@@ -0,0 +1,111 @@
+from app import api, db
+from app.auth.permissions import AdminOnly
+from app.datasets.models import Dataset
+
+from flask import request
+from sqlalchemy import asc
+from flask_restful import Resource
+from marshmallow import ValidationError
+from flask_jwt_extended import jwt_required, get_jwt_identity
+
+from .models import OwnershipRequest
+from .schemas import ownership_requests_schema, ownership_request_schema
+
+"""
+Routes to:
+- Create ownership request: OwnershipRequestView
+- View all ownership requests: AdminAllOwnershipRequestView
+- Approve or deny ownership requests: AdminOwnershipRequestView
+"""
+
+class OwnershipRequestView(Resource):
+    """ Creates ownership requests  """
+
+    @jwt_required
+    def post(self, _id):
+        """ Anyone who is logged in """
+
+        req_body = request.get_json()
+        req_body['dataset'] = _id
+
+        try:
+            new = ownership_request_schema.load(req_body)
+        except ValidationError as err:
+            return {'errors': err.messages}
+
+        # TODO: send email?
```

Comment:
Yes. I think sending an email to the admin that the ownership request would be helpful.

Reply:
Ok will move this to a new issue for another PR